example/.gitignore

```diff
@@ -23,6 +23,7 @@
 # Flutter/Dart/Pub related
 **/doc/api/
 **/ios/Flutter/.last_build_id
+**/ios/**/Pods/
```

Comment:
Do we want to move this to the repository level `.gitignore` rather than keep it within the example folder?

Reply:
@Victorrent I am planning on trying to contribute this to pub.dev, they have the check on the higher level .gitignore but not in here